src/application/scene/fxml/Drawboard.fxml

```diff
@@ -1,10 +1,123 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import java.lang.String?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.canvas.Canvas?>
 <?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ColorPicker?>
+<?import javafx.scene.control.Slider?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.control.ToggleButton?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.image.ImageView?>
 <?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.layout.BorderPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Pane?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.paint.Color?>
+<?import javafx.scene.text.Font?>
 
-<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" fx:controller="application.scene.Drawboard">
+<AnchorPane prefHeight="810.0" prefWidth="1315.0" style="-fx-background-color: grey;" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" fx:controller="application.scene.Drawboard">
    <children>
-      <Button fx:id="button_dashboard" layoutX="274.0" layoutY="188.0" mnemonicParsing="false" onAction="#dashboard" text="dashboard" />
+      <BorderPane prefHeight="810.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0">
+         <top>
+            <AnchorPane maxHeight="60.0" prefHeight="60.0" styleClass="navigation_bar" stylesheets="@../css/Drawboard.css" BorderPane.alignment="CENTER">
+               <children>
+                  <HBox prefHeight="60.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="350.0">
+                     <children>
+                        <Button fx:id="buttonDashboard" graphicTextGap="20.0" minHeight="-Infinity" minWidth="-Infinity" mnemonicParsing="false" onAction="#dashboard" prefHeight="60.0" prefWidth="200.0" styleClass="button_tool" stylesheets="@../css/Drawboard.css" text="dashboard">
+                           <graphic>
+                              <ImageView fitHeight="30.0" fitWidth="30.0" pickOnBounds="true" preserveRatio="true">
+                                 <image>
+                                    <Image url="@../assets/icon_dashboard_white.png" />
+                                 </image>
+                              </ImageView>
+                           </graphic>
+                        </Button>
+                        <Region minWidth="0.0" prefHeight="60.0" prefWidth="197.0" />
+                        <TextField fx:id="textFieldTitle" alignment="CENTER" prefHeight="40.0" prefWidth="500.0" styleClass="text_field_title" stylesheets="@../css/Drawboard.css" text="MY DRAWING (CLICK TO CHANGE TITLE)">
```

Comment:
could this be changed to placeholder text? that way it'll be deleted automatically when you click into the textbox rather than having to highlight it all and delete it

Reply:
The prompt text was updated to placeholder text in the fxml file